pyglmnet/base.py

```diff
@@ -1,171 +0,0 @@
-import warnings
```

Comment:
please don't remove this. We don't want a dependency on sklearn. It's a conscious choice.